docs/resources/deployment.md

```diff
@@ -0,0 +1,67 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "juju_deployment Resource - terraform-provider-juju"
+subcategory: ""
+description: |-
+  A resource that represents a Juju deployment.
+---
+
+# juju_deployment (Resource)
+
+A resource that represents a Juju deployment.
+
+## Example Usage
+
+```terraform
+resource "juju_deployment" "this" {
+  name = "foobar" # optional, set to charm name when absent
+
+  model = juju_model.development.uuid # required, model uuid
+
+  charm {
+    name     = "hello-kubecon" # required, supports CharmHub charms only
+    revision = ""              # optional, default: -1
+    channel  = ""              # optional, default: stable
+  }
```

Comment:
The `series` channel is missing. This parameter sets the Ubuntu version required to run the charm. For example, `juju deploy mysql --series=trusty`

Reply:
By default the field should be empty.